### PR TITLE
Added perl package from cpan

### PR DESCRIPTION
**Pull request by matthdsm**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

dependency for latest Ensembl VEP

### CODE REVIEW

File: recipes/perl-file-copy-recursive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  build:
+    - perl-threaded
+    - perl-pathtools
+
+  run:
+    - perl-threaded
+    - perl-pathtools
+
+test:
+  # Perl 'use' tests
```
Comment (bgruening) [inline recipes/perl-file-copy-recursive/meta.yaml]: Can you add a test?

**Comment by matthdsm** [comment]
it seems I issued this pull request way too early...
there is another perl dependecy short to build this package, but building this dependency creates a strew of other dependencies that aren't available in the bioconda channel. Should I submit a recipe for for each and every dependency that isn't available of is there another solution?

M

**Comment by bgruening** [comment]
You can submit everything in one PR. Please note that the skeleton command has a `--recursive` feature.

**Comment by leipzig** [comment]
looks like perl-file-copy-recursive did not get merged. Were there more issues with this?

**Pull request closed without merging**